lib/elixir/src/elixir_parser.yrl

```diff
@@ -256,6 +257,14 @@ access_expr -> bit_string : '$1'.
 access_expr -> sigil : build_sigil('$1').
 access_expr -> max_expr : '$1'.
 
+%% Augment integer literals with representation format if wrap_literals_in_blocks option is true
+number -> char : handle_literal(?exprs('$1'), '$1', [{format, char}]).
+number -> binary : handle_literal(?exprs('$1'), '$1', [{format, binary}]).
+number -> octal : handle_literal(?exprs('$1'), '$1', [{format, octal}]).
+number -> decimal : handle_literal(?exprs('$1'), '$1', [{format, decimal}]).
+number -> hex : handle_literal(?exprs('$1'), '$1', [{format, hexadecimal}]).
```

Comment:
Yeah, I know, but it is always used alongside decimal, octal and friends, so you can quickly guess the context from the surroundings. We can always change later if we find it confusing in future changes.